<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventCounterClass</class>
 <widget class="QDialog" name="EventCounterClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>246</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>352</width>
    <height>246</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>352</width>
    <height>246</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Event Counter</string>
  </property>
  <widget class="QFrame" name="bkgrndImage">
   <property name="geometry">
    <rect>
     <x>194</x>
     <y>-7</y>
     <width>171</width>
     <height>58</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>43</y>
     <width>321</width>
     <height>191</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QListWidget" name="cntrValueList">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>46</y>
      <width>171</width>
      <height>135</height>
     </rect>
    </property>
    <property name="flow">
     <enum>QListView::TopToBottom</enum>
    </property>
    <property name="isWrapping" stdset="0">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>204</x>
      <y>40</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStop">
    <property name="geometry">
     <rect>
      <x>204</x>
      <y>90</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConfig">
    <property name="geometry">
     <rect>
      <x>204</x>
      <y>140</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Configure</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="curValueEditor">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>10</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
